docs/OED_validation_guidelines.md

```diff
@@ -0,0 +1,84 @@
+
+
+### OasisLMF 1.23-LTS and later
+The CondTag field in the location file identifies the subset of locations to which a policy condition applies. 
+
+The account file contains the CondTag field, which links to the CongTag in the location file, and a CondNumber field which represents a particular set of financial terms for the condition. For each policy in the account file, one or more CondTags may be specified along with a CondNumber. More than one pair of CondTag, CondNumber values per policy gives rise to valid duplicates of the policy record. The financial terms for a given CondNumber must match if repeated on multiple rows. Each CondNumber is assigned a CondPriority which is the order in which the fiancial terms apply.
+
+If a location is assigned more than one CondTag in the location file, giving rise to duplicates, then the associated CondNumbers in the account file must have different CondPriorities. The terms are applied in order of CondPriority in this special case. There may not be multiple CondTags for a location in the location file which link to CondNumbers in the account file with the same CondPriority.
+
+## Cross validation
+If both OED location and account file are provided, the list of unique values of PortNumber, AccNumber must match between the files
+
+### OasisLMF 1.15.21-LTS
+If CondNumber is used in the location file for an account, each CondNumber value appearing in the location file must be appear for every PolNumber,LayerNumber under the same account in the account file.
+
+### OasisLMF 1.23-LTS and later
+If CondTag is used in the location file for an account, each CondNumber value appearing in the location file must be specified for at least one PolNumber,LayerNumber under the same account in the account file.
+
+## Step Policies
+A policy record in the account file is a step policy if data is populated in the StepNumber field and some of the other step fields (see 'OED Input Fields' within the [Open Exposure Data spec](https://github.com/OasisLMF/OpenDataStandards/blob/master/OpenExposureData/Docs/OpenExposureData_Spec.xlsx) with BackEndTableName 'StepFunctions' and 'Steps' for a full list). It is common to have multiple records in the account file for a step policy, with each record corresponding to each defined step in the policy.
+
+Step policies are incompatible with any other financial terms in OED account and location files in OasisLMF. Loc, Cond, Pol and Acc financial fields should not be populated for step policies. In particular, the CondNumber field must not be populated when StepNumber is populated, and vice versa StepNumber should not be populated where there are CondNumbers populated.
```

Comment:
you may add that step and non step policy can be mixed if they are in separate account
https://github.com/OasisLMF/OasisLMF/issues/829

Reply:
added